setup.py

```diff
@@ -17,9 +17,9 @@
         long_description = f.read()
 
     install_requirements = [
-        'vivarium==0.10.2',
+        'vivarium==0.10.4',
         'vivarium_public_health==0.10.9',
-
+        'vivarium_cluster_tools==1.2.3',
```

Comment:
Why'd you pull this up?

Reply:
I noticed lung cancer make_specs was failing with it down in the other block, so I moved it up. I forgot that make_specs is gone in this model though, so it should move back down...